network-store-client/src/main/java/com/powsybl/network/store/client/RestClientImpl.java

```diff
@@ -49,74 +46,94 @@ private <T extends IdentifiableAttributes> ResponseEntity<TopLevelDocument<T>> g
         return restTemplate.exchange(url,
                 HttpMethod.GET,
                 new HttpEntity<>(new HttpHeaders()),
-                new ParameterizedTypeReference<TopLevelDocument<T>>() {
+                new ParameterizedTypeReference<>() {
```

Comment:
this reverts https://github.com/powsybl/powsybl-network-store/commit/9744168f47210eab11796861f9dcf4ffdd5aea0c is it ok ?

Reply:
I was not aware of this. I revert.

Reply:
reverted